docs/common/adding-http-client-to-integration-studio.md

```diff
@@ -0,0 +1,10 @@
+# Inbuilt HTTP Client Guidelines
+
+WSO2 Integration Studio provides you with an inbuilt HTTP client that facilitates developers to test the integrations in the
+Integration Studio itself.
+
+1. Click on **Window** menu and select **Show View** > **Other...** option. 
```

Comment:
HTTP Client window is already available by default. Shall we add a screenshot to show where it is and add this step as an optional step if the window is not visible?

Reply:
fixed